libweb3jsonrpc/abstractwebthreestubserver.h

```diff
@@ -10,7 +10,7 @@
 class AbstractWebThreeStubServer : public jsonrpc::AbstractServer<AbstractWebThreeStubServer>
 {
     public:
-        AbstractWebThreeStubServer(jsonrpc::AbstractServerConnector &conn) : jsonrpc::AbstractServer<AbstractWebThreeStubServer>(conn)
+        AbstractWebThreeStubServer(jsonrpc::AbstractServerConnector &conn) : jsonrpc::AbstractServer<AbstractWebThreeStubServer>(&conn)
```

Comment:
I hadn't realized I made this change to make compatible with an older jsonrpc library. Will update accordingly.